src/github/pull-request.ts

```diff
@@ -84,7 +83,7 @@ export const pullRequestWebhookHandler = async (context: WebhookContext, jiraCli
 		return;
 	}
 
-	context.log.info(`Sending pull request update to Jira ${baseUrl}`);
+	context.log.info({ jiraHost : baseUrl }, `Sending pull request update to Jira`);
```

Comment:
can you plz tell me why move it up front?

Reply:
Moved it out of the template string and into to the optional fields object. The entries in fields object will be run through the sensitive data serializer, so it will hash jiraHost for us.
Also, its way way way easier on splunk to read if it has the values as attributes outside of the msg.